openhtf/core/test_executor.py

```diff
@@ -109,7 +109,7 @@ def finalize(self):
   def wait(self):
     """Waits until death."""
     try:
-      self.join(sys.float_info.max)  # Timeout needed for SIGINT handling.
+      self.join(5)  # Timeout needed for SIGINT handling.
```

Comment:
This is a question for the maintainers. In Python3, self.join raises a ValueError since sys.float_info.max is too large of a value to wait on.

This is only test code, and the timeout doesn't seem to be intended to be tested, so I put 5 seconds in as a non-float-max placeholder. Is this reasonable?